semantic_conventions/resource/host.yaml

```
@@ -9,7 +9,8 @@ groups:
         type: string
         brief: >
           Unique host ID. For Cloud, this must be the instance_id assigned by the cloud provider.
-        examples: ['opentelemetry-test']
+          For non-containerised Linux systems, the `machine-id` located in `/etc/machine-id` or `/var/lib/dbus/machine-id` may be used.
```

```suggestion
          For non-containerized Linux systems, the `machine-id` located in `/etc/machine-id` or `/var/lib/dbus/machine-id` may be used.
```
I think we mostly use American English throughout the spec, so this would keep it consistent.

---

Thx